huxley/www/templates/www.html

```diff
@@ -23,5 +23,9 @@
     {% include "ie-warning.html" %}
     {% include "js-warning.html" %}
     <div id="huxley-app"></div>
+    <div style="background-color: #111; position: fixed; bottom: 0pt; right: 0pt; text-align: right; font-weight: 600; padding: 5px 15px 5px 10px; background-color: rgba(17, 17, 17, 0.6); min-width: 135px; border-top-left-radius: 6px">
```

Comment:
To clean this up a bit, I would put the styling for each of these elements into one css file.